### DISCUSSION TITLE
[A-L] (2023/24) Foglio 7 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#177] [Esercizi - Bertini]
![Screenshot 2023-11-13 194608](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/d306ef39-fc82-4552-bc85-37aeedf84b82)

**Comment by Elia-Belli** [comment]
Il dado è equo quindi ogni faccia ha $1/6$ di probabilità di uscire, definiamo l'evento $A=\\{esce\ 6\ o\ 5\\}$ con $\mathbb{P}(A)=p=1/3$.
- **(1)** I lanci arrivano a $3$ se $A$ avviene per la prima volta al 3° lancio, quindi $\mathbb{P}(T=3)=(1-p)^2\cdot p$, se vogliamo calcolare la probabilità $\mathbb{P}(T=3,X=5)$ escludiamo l'uscità di $5,6$ nei primi due lanci e quella di tutti tranne il $5$ nell'ultimo: $\mathbb{P}(T=3,X=5)=(1-p)^2\cdot 1/6=(2/3)^2\cdot 1/6=2/27$
- **(2)** Notiamo che $T\sim Geom(p)$ allora $\mathbb{P}(T=t)=(1-p)^{t-1}\cdot p$
- **(3)** $X$ è una v.a. binaria, se ci troviamo all'ultimo lancio il risultato è stato $5$ o $6$ equamente, allora $\mathbb{P}(X=x)=1/2$
- **(4)** Sia $\mathbb{P}(T=t)$ la probabilità di terminare i lanci al t-esimo e $X$ il risultato del t-esimo lancio:
$\mathbb{P}(T=t,X=x)=\mathbb{P}(T=t)\cdot \mathbb{P}(X=x)$ infatti, come al punto **(1)** $(1-p)^2\cdot 1/6=[(1-p)^2\cdot 1/3]\cdot 1/2$,
in generale $(1-p)^{t-1}\cdot 1/6=[(1-p)^{t-1}\cdot 1/3]\cdot 1/2,\forall t\in \mathbb{N}^+$, allora $T,X$ sono indipendenti.

**Comment by CiottoloMaggico** [comment]
Analogo @Elia-Belli 
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/0d8567de-6640-4425-99ef-2199bdf0f4fb)
